encoder/tests/interfaces

```diff
@@ -60,6 +60,9 @@ export interface I2 {
   I2_OptionalInterfaceReference?: I1;
   I2_TypedList: number[];
 }
+export const i2Default: I2 = {
```

Comment:
we can see the default for list is generated all the time into type script, because of the following:
```
	var r cue.Runtime
	i, err := r.Compile("testcases1.cue", `BType: [...int]`)
	if err != nil {
		log.Fatal(err)
	}

	v, ok := i.Value().Lookup("BType").Default()
	fmt.Printf("the default value %v, %v", ok, v)
```

The default value for list is by default [], It could make sense even it is not really what we want exactly. From typescript point of view, it should have no difference?

Reply:
this need to be correct by Cuetsy intent to remove the false default for the list.